sdk/eventhub/azure-eventhub-checkpointstoreblob/azure/eventhub/extensions/checkpointstoreblob/_blobstoragecs.py

```diff
@@ -10,7 +10,7 @@
 from datetime import datetime
 from collections import defaultdict
 
-from azure.eventhub import CheckpointStore  # type: ignore  # pylint: disable=no-name-in-module
+from azure.eventhub._eventprocessor.checkpoint_store import CheckpointStore  # type: ignore  # pylint: disable=no-name-in-module
```

Comment:
Since `CheckpointStore` is part of the public API, we don't need to import from private files here. You can revert this to how it was.